setup.py

```diff
@@ -23,5 +23,5 @@
         'Programming Language :: Python :: 3.4',
     ],
     test_suite='runtests.runtests',
-    tests_require=['responses>=0.5']
+    tests_require=['responses>=0.5', 'mock>=1.3']
```

Comment:
A very good point! I shall do that; is checking `sys.version_info < (3, 4)` the best way to do that?